src/client/game/index.js

```diff
@@ -20,9 +20,9 @@ export default class Game {
       this.pointerMoved = this.pointerMoved.bind(this);
       window.addEventListener('pointermove', this.pointerMoved);
       window.addEventListener('pointerdown', this.pointerMoved);
-      this.resizeHandler = (e => this.canvasHeight = this.app.view.clientHeight).bind(this);
+      this.canvasHeight = document.body.clientHeight;
+      this.resizeHandler = (e => this.canvasHeight = this.element.clientHeight).bind(this);
```

Comment:
This won't work if we make the canvas `object-fit: contain;` in the future. I think the right fix here would be to not call the resize handler in the constructor but expose a method on the game object that will set the `canvasHeight` and call it on mount. (and if for some reason the canvas is moved/resized without the window being resizes)

as a side note, I missed this when it was first merged but you don't need to bind arrow functions to `this` since they will capture the this from the scope where they're declared, the first argument passed to bind on arrow functions is actually ignored. You can still call bind with more than one argument for partial application.